internal/install/recipe_installer.go

```diff
@@ -130,45 +140,40 @@ func (i *RecipeInstaller) Install() error {
 		}
 
 		for _, r := range recipes {
-			log.Debugf(fmt.Sprintf("Found available integration %s.", r.Name))
+			log.Debugf("Found available integration %s.", r.Name)
 		}
 
 		i.reportRecipesAvailable(recipes)
 	}
 
-	log.Debugf(fmt.Sprintf("AssumeYes:%t.", i.AssumeYes))
-	log.Debugf(fmt.Sprintf("RecipeNames:%s.", i.RecipeNames))
-	log.Debugf(fmt.Sprintf("RecipePaths:%s.", i.RecipePaths))
-	log.Debugf(fmt.Sprintf("SkipDiscovery:%t.", i.SkipDiscovery))
-	log.Debugf(fmt.Sprintf("SkipInfraInstall:%t.", i.SkipInfraInstall))
-	log.Debugf(fmt.Sprintf("SkipLoggingInstall:%t.", i.SkipLoggingInstall))
-	log.Debugf(fmt.Sprintf("SkipIntegrations:%t.", i.SkipIntegrations))
-	log.Debugf(fmt.Sprintf("RecipesProvided:%t.", i.RecipesProvided()))
+	log.Debugf("InstallerContext: %+v\n", i.InstallerContext)
```

Comment:
Can we keep the debug statement for the output of the method i.RecipesProvided() ?

Reply:
👍